vocab_list/models.py

```diff
@@ -222,7 +231,8 @@ class Meta:
         verbose_name = "personal vocabulary list entry"
         verbose_name_plural = "personal vocabulary list entries"
         order_with_respect_to = "vocabulary_list"
-        unique_together = ("vocabulary_list", "headword")
+        # see comment for VocabularyListEntry
+        # unique_together = ("vocabulary_list", "headword")
```

Comment:
Same comment here about deleting `unique_together`. A comment probably isn't needed here if the assumptions are documented in the base class.